com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/graph/ResourceGraphs.java

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020-2023 The Defold Foundation
+// Copyright 2014-2020 King
+// Copyright 2009-2014 Ragnar Svensson, Christian Murray
+// Licensed under the Defold License version 1.0 (the "License"); you may not use
+// this file except in compliance with the License.
+// 
+// You may obtain a copy of the License, together with FAQs at
+// https://www.defold.com/license
+// 
+// Unless required by applicable law or agreed to in writing, software distributed
+// under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+// CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+
+package com.dynamo.bob.pipeline.graph;
+
+import java.util.Stack;
+import java.util.Set;
+import java.util.HashSet;
+
+import com.dynamo.bob.Project;
+import com.dynamo.bob.CompileExceptionError;
+import com.dynamo.bob.fs.IResource;
+import com.dynamo.bob.pipeline.graph.ResourceWalker;
+import com.dynamo.bob.pipeline.graph.ResourceWalker.IResourceVisitor;
+
+
+public class ResourceGraphs {
+
+    private ResourceGraph liveUpdateResourceGraph;
+    private ResourceGraph fullResourceGraph;
```

Comment:
I have some thought about our upcoming changes.

1. we want multiple output archives, and they won't  really be graphs as such, but rather just a collection of files matching certain filename patterns.
I guess it should be easy to remove this liveupdate graph then?

2. our current two graphs overlap (e.g. a file in the LU tree may exist in the full graph.
The .json file will show the full graph only, so I'm wondering what the use of the liveupdate graph really is (of this specific class)? Isn't it enough to create just one graph?

Reply:
It's not a big deal for me, I assume this will be reworked in the next iteration of LU.